### PR TITLE
Prepare for release 1.1.0 "Black Crow"

### PR DESCRIPTION
**Pull request by Leon-Bichmann**
Updates since 1.0.0:

- mhcflurry conda package added
- mhcflurry predictions added
- peak picking preprocessing added
- minor changes in default parameters
- few arguments added eg. min_peptide_len, max_peptide_len

## PR checklist
 - [x] This comment contains a description of changes (with reason)
 - [x] If you've fixed a bug or added code that should be tested, add tests!
 - [x] If necessary, also make a PR on the [nf-core/mhcquant branch on the nf-core/test-datasets repo]( https://github.com/nf-core/test-datasets/pull/new/nf-core/mhcquant)
 - [x] Ensure the test suite passes (`nextflow run . -profile test,docker`).
 - [x] Make sure your code lints (`nf-core lint .`).
 - [x] Documentation in `docs` is updated
 - [ ] `CHANGELOG.md` is updated
 - [x] `README.md` is updated

**Learn more about contributing:** https://github.com/nf-core/mhcquant/tree/master/.github/CONTRIBUTING.md

### CODE REVIEW

**Comment by Leon-Bichmann** [comment]
So it seems like travis wait did not work, Im not sure whats the problem with travis here

File: Dockerfile [hunk 1]
```diff
@@ -8,3 +8,4 @@ RUN conda env create -f /environment.yml && conda clean -a
 RUN conda env create -f /environment-percolator.yml && conda clean -a
 ENV PATH /opt/conda/envs/nf-core-mhcquant-1.1.0/bin:$PATH
 ENV PATH /opt/conda/envs/nf-core-mhcquant-percolator-1.0dev/bin:$PATH
+RUN mhcflurry-downloads fetch models_class1
```
Comment (apeltzer) [inline Dockerfile]: This will increase your container size quite substantially :-(

File: main.nf [hunk 2]
```diff
@@ -252,21 +252,6 @@ ${summary.collect { k,v -> "            <dt>$k</dt><dd><samp>${v ?: '<span style
 }
 
 
-/*
```
Comment (apeltzer) [inline main.nf]: Why not keep this and provide an extra parameter that enables users to specify the path to already prefeteched models? This way, we'd have slim containers, flexibility _and_ the possibility to download models if we require them :-)

**Comment by Leon-Bichmann** [comment]
As the tests pass now, I would maybe release it as it is now - so we have a working version running. Im happy to adapt the Dockerfile at a later stage if I find out how...

File: main.nf (hunk 2 above)
Comment (apeltzer) [reply main.nf]: That's weird - do you have the code for that available somewhere? Shouldn't be a big issue to adapt it in a way to keep this out of the container...

**Comment by apeltzer** [comment]
Looking good except for the download part - can we please figure out whats going on there instead of moving the download in the container itself?

File: main.nf (hunk 2 above)
Comment (Leon-Bichmann) [reply main.nf]: hmm.. i mean you can try it yourself when you remove the line 'fetch' from the dockerfile and add the process instead the final process 'predict_peptides' which depends on this download will fail.

Comment (apeltzer) [reply main.nf]: I just added it to the `predict_peptides`step itself to see what happens 👍

**Comment by apeltzer** [comment]
Can you document what you did in the CHANGELOG, please? And maybe add some more documentation as mentioned in https://github.com/nf-core/mhcquant/issues/15 ? 

At least the CHANGELOG would be nice and then some more documentation on what the output files mean etc... so that people can interpret what the output is all about :-)

File: main.nf (hunk 2 above)
Comment (Leon-Bichmann) [reply main.nf]: ok I see - well great that it works.

**Comment by Leon-Bichmann** [comment]
alright ill try to take care of this tomorrow thanks so far already!

**Comment by apeltzer** [comment]
Perfect - we’ll get this shipped tomorrow then :)

File: environment.yml [hunk 3]
```diff
@@ -4,6 +4,7 @@ channels:
   - conda-forge
   - defaults
 dependencies:
+  - conda-forge::r-rmarkdown=1.10
```
Comment (Leon-Bichmann) [inline environment.yml]: Is this sufficient as R dependency?

**Comment by Leon-Bichmann** [comment]
`ERROR ~ Error executing process > 'output_documentation'
Caused by:
  Process `output_documentation` terminated with an error exit status (127)
Command executed:
  markdown_to_html.r output.md results_description.html
Command exit status:
  127
Command output:
  (empty)
Command error:
  /usr/bin/env: ‘Rscript’: No such file or directory`

File: environment.yml (hunk 3 above)
Comment (apeltzer) [reply environment.yml]: Should be :-)

**Comment by apeltzer** [comment]
> `ERROR ~ Error executing process > 'output_documentation' Caused by: Process `output_documentation` terminated with an error exit status (127) Command executed: markdown_to_html.r output.md results_description.html Command exit status: 127 Command output: (empty) Command error: /usr/bin/env: ‘Rscript’: No such file or directory`

This is something that we'd just need to retrigger on Travis CI as the docker image wasn't built already when you submitted the Git commit here...

**Comment by apeltzer** [comment]
After retriggering, this ran through fine - I'M going to merge and release now.

**Pull request merged**